src/Display.tsx

```diff
@@ -218,39 +223,29 @@ const VictimDetail: React.FC<VictimDetailProps> = props => {
 
 interface DisplayListProps {
   isReady: boolean;
-  responseOk: boolean;
+  responseOk: any;
   dataArray: any;
 }
 
 const DisplayList: React.FC<DisplayListProps> = props => {
   let content: any;
-  if (props.dataArray.length === 0) {
-    if (props.isReady === false) {
-      content = <Alert variant="danger" isInline title="No data available" />;
-    } else {
-      content = null;
-    }
+  if (props.dataArray == undefined) {
+    content = "";
+  } else if (props.dataArray.length === 0) {
+    content = <Alert variant="danger" isInline title="No data available" />;
   } else {
     content = props.dataArray.map((val: any, key: number) => (
       <li key={key}>
         <VictimDetail data={val.map}></VictimDetail>
       </li>
     ));
   }
-  if (!props.responseOk) {
-    content = (
-      <Alert
-        variant="danger"
-        isInline
-        title="Error: Emergency Response services unreachable"
-      />
-    );
-  }
```

Comment:
Because there is no proper response for unreachable service error from findmyrelative API.
Response is same for "no data" and 503 when we try to hit Emergency response demo api and when it's down.

Proper status response is needed in order to differentiate between findmyrelative service and emergency response demo service unreachable error.

Once API response comes as a text with status. We can un-comment it.

I have completed it prior so that dependency on the same won't come.